test/gtest/gtest_common.hpp

```diff
@@ -114,6 +114,28 @@ bool IsTestSupportedForDevMask()
     return res;
 }
 
+inline Gpu GetDevGpuType()
```

Comment:
[optimization] Rename to `GetDevGpuType_impl` and add 
```C++
Gpu GetDevGpuType()
{
    static const auto rv = GetDevGpuType_impl();
    return rv;
}
```
or use lambda. This would avoid repeating of string comparisons.